lib/plottr_components/src/components/timeline/TimelineTable.js

```diff
@@ -32,6 +32,7 @@ const TimelineTableConnector = (connector) => {
     state = {
       tableLength: 0,
       hovering: null,
+      draggedBeat: {}
```

Comment:
Are you sure that this is the best way to encode that there's no beat being dragged currently?
If I wanted to check that a beat was being dragged, I'd need to check for an empty attribute on this object rather than checking whether there is an object at all.
(I'd recommend using `null` instead here).